<launch>

  <node pkg="navgraph_solver" type="nav_move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find teb_local_planner_tutorials)/cfg/diff_drive/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find teb_local_planner_tutorials)/cfg/diff_drive/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find teb_local_planner_tutorials)/cfg/diff_drive/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find teb_local_planner_tutorials)/cfg/diff_drive/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find teb_local_planner_tutorials)/cfg/diff_drive/teb_local_planner_params.yaml" command="load" />

    <param name="base_global_planner" value="navgraph_solver/NavgraphGlobalPlanner" />
    <param name="global_planner" value="global_planner/GlobalPlanner" />
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
    <param name="points_file" value="/home/divel2/catkin_ws/src/navgraph_solver/navgraphs/points_rviz.csv"/>
    <param name="adjacency_file" value="/home/divel2/catkin_ws/src/navgraph_solver/navgraphs/adjacency_rviz.csv"/>
    <param name="lethal_cost" value="255" />
    <param name="neutral_cost" value="1" />
    <param name="cost_factor" value="0.6986" />
    <param name="use_grid_path" value="false" />
    <param name="planner_frequency" value="0.0" />
    <param name="planner_patience" value="5.0" />
    <param name="controller_frequency" value="5.0" />
    <param name="controller_patience" value="15.0" />
  </node>

</launch>